package.json

```diff
@@ -90,6 +90,8 @@
   },
   "scripts": {
     "clean": "rm -rf node_modules && rm -f package-lock.json && npm install",
+    "install-driver": "appium driver install --source=local $(pwd)",
+    "reinstall-driver": "(appium driver uninstall xcuitest || exit 0) && appium driver install --source=local $(pwd)",
```

Comment:
These additions can be removed. I wasn't sure best way to test changes and having these scripts were handy for quickly reinstalling driver

Reply:
yes, please remove them. These additions are unrelated to the PR topic

Reply:
Removed these additions